scripts/configure.py

```diff
@@ -85,6 +85,8 @@
              'xkcdpass',                    # Random passphrase generation
              'systemd-journal-remote',      # Remote/web based log access
              'jq',                          # JSON parser used in check-release script
+             # pygobject dependencies (Spotifyd)
+             'libgirepository1.0-dev', 'libcairo2-dev'
```

Comment:
Looks like you will need to add these to `.github/workflows/python-app.yml` as well